scanner.py

```diff
@@ -2494,6 +2498,9 @@ def save(self, scanner, show_message=True):
         configfile.set(section, "model_temp", "%f" % (self.temp))
         configfile.set(section, "model_offset", "%.5f" % (self.offset,))
         configfile.set(section, "model_mode", "%s" % (self.scanner.calibration_method))
+        configfile.set(section, "model_created_on", str(timestamp))
+        configfile.set(section, "model_mcu_fw", str(mcufw))
+        configfile.set(section, "model_scanner_version", str(SCANNER_VERSION))
```

Comment:
```suggestion
        configfile.set(section, "model_software_version", str(SCANNER_VERSION))
```

Reply:
hmm, i still think people will confuse this with klipper version and scanner version somehow?

Reply:
Why? Also this is for us, not for users.